asyncapi-2025-goal.md

```diff
@@ -0,0 +1,98 @@
+Carrying on from Thulie, it's time to have an open discussion on the goals for building and maintaining the AsyncAPI community. The Community Manager needs to ensure the goals are addressed and that each program/initiative has a dedicated person responsible for it.
+
+Things I think we should focus on in 2025:
+
+- Increase Maintainers
+- Onboarding Docs and Video Content
+- Community Working Group
+
+Things we should continue investing in:
+
+- Community Initiatives and Programs
+
+## Increase Maintainers
+Increasing the number of maintainers in our community remains one of our top goals, and we should continue to explore and experiment with strategies to achieve this. We have three strategies that could contribute significantly to our long-term goal of growing the number of maintainers.
+
+1. **Fully Implement Maintainership** - We are currently in the pilot stages of implementing maintainership and are already seeing positive outcomes. I believe that by 2025, we need to fully integrate this into all our programs. The program will cater to anyone looking to enhance their skills as maintainers and to learn what it takes to become an open-source maintainer. Additionally, we should be open to individuals who want to apply and don't require any stipend.
+
+2. **AsyncAPI Grant Program** - The AsyncAPI Community Grant Program, funded by the AsyncAPI Initiative, aims to provide financial support for critical technical projects and community initiatives within AsyncAPI. This initiative directly contributes to the ongoing goal of growing and retaining maintainers while sustaining the community. The AsyncAPI Grant Program will specifically cater to individual contributors who are part of TSC members and maintainers.
+
+3. **Increase Corpo Involvement** - 
```

Comment:
@derberg, I forgot what I'm supposed to add here 🙈

Reply:
it was about increasing business involvement - to have them more maintainers from that side

we also talked about this vendors ecosystem on marketing call - how to get vendors/sponsors more involved in maintainance - not necessarily code only